runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkJobInvocation.java

```diff
@@ -133,8 +133,8 @@ PipelineResult runPipelineWithTranslator(FlinkPortablePipelineTranslator<T> tran
     RunnerApi.Pipeline fusedPipeline =
         trimmedPipeline.getComponents().getTransformsMap().values().stream()
                 .anyMatch(proto -> ExecutableStage.URN.equals(proto.getSpec().getUrn()))
-            ? pipeline
-            : GreedyPipelineFuser.fuse(pipeline).toPipeline();
+            ? trimmedPipeline
+            : GreedyPipelineFuser.fuse(trimmedPipeline).toPipeline();
```

Comment:
Thanks for spotting. Looks like a bug caused by rebasing to the latest master.

CC @robertwb